src/pages/pages/integrations.md

```diff
@@ -0,0 +1,44 @@
+---
+templateKey: default-page
+seo:
+  description: "Integrations for Ironic Bare Metal"
+  title: Ironic Bare Metal Integrations
+  twitterUsername: "@openstack"
+  url: "https://ironicbaremetal.org/"
+title: Integrations
+subTitle: "Ironic project Integrations and Tools"
+---
+
+The Ironic project has a long history of being driven by Cloud and Infrastruture operators seeking to solve their problems. This has resulted in a diverse set of capabilities, integrations, and ways to leverage ironic.
+
+<br>
+
+## Community Projects with Integration
+
+This is a list of community projects which feature or leverage Ironic in order to deploy Bare Metal to meet the needs of infrastucture operators and ultimately end users. Some of these projects have commercial versions or offerings from vendors, however this list is restricted to "Community" only.
+
+- **Bifrost:** <https://docs.openstack.org/bifrost/latest/install/index.html> - An Ansible based toolkit to setup Ironic, and orchustrate Bare Metal with Playbooks!
```

Comment:
"orchestrate", "Bare Metal provisioning"

Reply:
Change made in working copy.